src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -88,6 +90,11 @@ public ResponseEntity<Object> createNewSubmission(
         }
     }
 
+    private String convertErrorsToString(Errors validationErrors) {
+        var gson = new GsonBuilder().create();
+        return gson.toJson(validationErrors);
```

Comment:
I'm interested in what this looks like in the output. Could you please copy and paste an extract from the Docker Tilt log here?

Also, minor, but suggest renaming method to `convertErrorsToJsonString` - conveys the type of string you'll get back.

Reply:
An example can be seen above.

Reply:
From the logs:
{"created":"2022-10-25T09:23:16.310Z","event":"error","namespace":"overseas-entities-api","context":"NviWdOblk31O714ygPYjs8APsar8","data":{"message":"Validation errors : {\"errs\":[{\"error\":\"Argentina\\\\u0414\\\\u0440\\\\u0430\\\\u043a\\\\u043e\\\\u043d is not on the list of allowed countries\",\"location\":\"entity.incorporation_country\",\"locationType\":\"request-body\",\"type\":\"ch:validation\"}]}"}}

Reply:
👍 nice. Thanks.